Successfully updated:
Created Frame node pR_DJKJgp
Created Frame node p2YUvcEEV
Created Text node jJwxhTSsO
Created Text node l4CIU3ZFR
Created Frame node LcNPrqe5B
Updated attributes for node pR_DJKJgp
Updated attributes for node p2YUvcEEV
Updated text for node jJwxhTSsO
Updated attributes for node jJwxhTSsO
Updated text for node l4CIU3ZFR
Updated attributes for node l4CIU3ZFR
Updated attributes for node LcNPrqe5B
Reordered node p2YUvcEEV within parent pR_DJKJgp to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -2976,20 +2976,73 @@
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   Body content goes here
 </BodyContentGoesHere>
 <Something
     nodeId="so_W61aYl"
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   something
 </Something>
+<Stack
+    nodeId="pR_DJKJgp"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="p2YUvcEEV"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="jJwxhTSsO"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="l4CIU3ZFR"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="LcNPrqe5B"
+      width="100%"
+      height="100px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>


Note: Set zoomIntoView=false if you want to use Framer app while MCP is working.